Sources/PostgresNIO/New/PSQLRowSequence.swift

```diff
@@ -0,0 +1,609 @@
+import NIOCore
+import NIOConcurrencyHelpers
+
+#if swift(>=5.5) && canImport(_Concurrency)
+/// An async sequence of ``PSQLRow``s.
+///
+/// - NOTE:This is a struct to allow us to move to a move only type easily once they become available.
```

Comment:
I am legally (though IANAL) obligated to offer this utterly irrelevant correction:
```suggestion
/// - Note: This is a struct to allow us to move to a move only type easily once they become available.
```